resolwe_bio/processes/workflows/wes.yml

```diff
@@ -45,6 +45,20 @@
       required: true
       description: |
         dbSNP database of variants for variant calling.
+    - name: validation_stringency
+      label: Validation stringency for all SAM files read by this program. Setting stringency to SILENT
+        can improve performance when processing a BAM file in which variable-length data (read,
+        qualities, tags) do not otherwise need to be decoded. Default is STRICT. This setting is used
+        in BaseRecalibrator and ApplyBQSR processes.
+      type: basic:string
+      default: STRICT
+      choices:
+        - label: STRICT
```

Comment:
Use sentence case.

Reply:
I would prefer keeping things as they appear in the tool. That way there is less confusion when checking the tool manual and comparing it to this entry.